src/args.rs

```diff
@@ -75,7 +75,7 @@ pub struct InitArgs {
 }
 impl InitArgs {
     pub fn get_target_language(&self) -> TargetLanguage {
-        self.language.unwrap_or({
+        self.language.unwrap_or_else( || {
```

Comment:
The formatter is complaining
```suggestion
        self.language.unwrap_or_else(|| {
```